src/Model/Endpoint.php

```diff
@@ -659,6 +661,9 @@ public function findAll(Query $query, array $options): Query
      */
     public function findList(Query $query, array $options): Query
```

Comment:
This method needs to be updated similar to core's `Table::findList()`

Reply:
Seeing that the findList method in core returns a SelectQuery. Do you want me to implement the different query types core has here as well? Or just the way this method works? @ADmad